metecho/api/jobs.py

```diff
@@ -146,6 +154,158 @@ def _create_branches_on_github(
     return task_branch_name
 
 
+def get_orgs_for_user(user: User):
+    """
+    Filter the local GitHubOrganizations down to those that the user is a member of.
+    """
+
+    try:
+        gh_user = gh_given_user(user)
+        qs = GitHubOrganization.objects.filter(
+            login__in=(org.login for org in gh_user.organizations())
+        )
+        serialized = [{"id": str(org.id), "name": org.name} for org in qs]
```

Comment:
@jerivas Thinking about this more, I'm wondering if we could store this data in Metecho and add an endpoint to fetch this list without necessarily queuing a job. That would more closely align with how we handle GH repos. The idea would be that the list in Metecho would be updated periodically in the background (on login), but then the cached list could be fetched by the FE any time. (Right now, there's no way for the FE to get the list without either getting the _full_ list of Orgs, or triggering a new job.) What do you think?

Reply:
That sounds promising. I'm thinking we can add a new m2m field that maps local Users to local GitHubOrganizations, then update the mapping on login. Would we need a `currently_refreshing_orgs` flag on the User?

Reply:
That makes sense to me. I guess adding `currently_refreshing_orgs` is probably a good idea -- if we're caching, we probably need to allow users to explicitly refresh the cache.

Reply:
In a0f07f1a4eb27e197baa18a878ad20c9b1e05861 I added an `organizations` field to the User model. This will be part of the regular User payload at all times and can be refreshed on demand just like the repos (POST to `/api/user/refresh_orgs/`). Right now the refresh job doesn't include the list of fresh orgs in the WS message. Is this something I should add?

Reply:
> Right now the refresh job doesn't include the list of fresh orgs in the WS message. Is this something I should add?

We'll probably want that, but let me dig into FE implementation first -- I'll let you know if I need non-trivial changes. 👍